File: DevSupportAndroid/src/main/java/com/sup/dev/android/libs/image_loader/ImageLoaderBytes.java
package com.sup.dev.android.libs.image_loader;

public class ImageLoaderBytes extends ImageLoaderA {

    private final byte[] bytes;

    public ImageLoaderBytes(Object key, byte[] bytes) {
        this.bytes = bytes;
        setKey("bytes_" + key);
    }

    byte[] load() {
        return bytes;
    }
}

File: src/main/java/com/sup/dev/android/libs/image_loader/ImageLoaderBytes.kt
package com.sup.dev.android.libs.image_loader

class ImageLoaderBytes(
        private val key: Any,
        private val bytes: ByteArray
) : ImageLoaderA() {

    override fun getKey() = "bytes_${key}_${w}_${h}"

    override fun load(): ByteArray {
        return bytes
    }
}